lang/cs/Org.Apache.REEF.Tang/Formats/AvroConfigurationDataContract/AvroConfiguration.cs

```diff
@@ -37,7 +38,7 @@ public AvroConfiguration()
         }
 
         [DataMember]
-        public HashSet<ConfigurationEntry> Bindings { get; set; }
+        public List<ConfigurationEntry> Bindings { get; set; }
```

Comment:
What is the reason to convert the set to list? Do you have duplicated entries in the configuration?